code/src/components/product-skeleton/ProductSkeleton.scss

```diff
@@ -6,6 +6,7 @@
 
   &__footer {
     display: flex;
+    width: 234px !important;
```

Comment:
Generally we add comment to inform that this is for overriding mui styles, because using `!important` is not a good practice. We discussed it with our team. Check how we did this in other places in the project (use global search in your IDE)